applications/asset_tracker/src/main.c

```diff
@@ -462,8 +466,19 @@ static void device_status_send(struct k_work *work)
 #endif
 	};
 
-	if (service_info_json_object_encode(ui, ARRAY_SIZE(ui), NULL, 0,
-			SERVICE_INFO_FOTA_VER_CURRENT, root_obj) == 0) {
+	const char *const fota[] = {
+#if IS_ENABLED(CONFIG_CLOUD_FOTA_APP)
```

Comment:
`IS_ENABLED` is typically used for runtime checks, while only `if` or `if defined` is sufficient here

Reply:
Replaced with `#if defined`